releasenotes/notes/support-trace-logging-for-python-checks-f72157a532eb3616.yaml

```diff
@@ -0,0 +1,11 @@
+# Each section from every releasenote are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+features:
```

Comment:
nit: I'd put this under `enhancements` instead, I'd like to keep `features` for "very significant" features 😃